backend/configuration/configuration_api.py

```diff
@@ -57,13 +58,12 @@ def clone_git_repo_and_store_path_in_database(git_repo_address, database, local_
     :return:                            {Dictionary}    {'success':True} if everything worked, {'success':False} otherwise
     """
     try:
-        controller = Controller(git_repo_address,
-                                os.path.dirname(os.path.abspath(__file__)) + os.getenv(local_repo_path_env_string))
+        controller = Controller(git_repo_address, Path(os.path.dirname(os.path.abspath(__file__)) + os.getenv(local_repo_path_env_string)))
```

Comment:
In general, I'd prefer shorter lines. ~79
That's more or less standard, see e.g. in Python [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)

Reply:
I will change this :)